src/sandbox/components/side-nav.tsx

```diff
@@ -49,23 +48,27 @@ export const SideNav: React.FC<StackProps> = props => {
     <Stack borderRight={border()} {...props}>
       {navigation.map(nav =>
         nav.isDisabled ? null : (
-          <Link href={nav.url}>
-            <Grid
-              borderRight={border()}
-              bg={color('bg')}
-              borderBottom={border()}
-              size="72px"
-              placeItems="center"
-              justifyContent="center"
-              color={color('text-title')}
-              _hover={{
-                cursor: 'pointer',
-                bg: color('bg-2'),
-              }}
-            >
-              {nav.icon}
-            </Grid>
-          </Link>
+          <Tooltip placement="left" label={nav.label}>
+            <div>
```

Comment:
< Tooltip /> tooltip round <Link /> doesn't render the tooltip, so I had to add the div.